test-network-function/identifiers/identifiers.go

```diff
@@ -577,6 +590,23 @@ the changes for you.`,
 			`,
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
 	},
+	TestReadinessIdentifier: {
+		Identifier: TestReadinessIdentifier,
+		Type:       normativeResult,
+		Description: formDescription(TestShudtownIdentifier,
+			`Ensure that readiness is defined.`),
+		Remediation:           ``,
+		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
+	},
+	TestLivenessIdentifier: {
+		Identifier: TestLivenessIdentifier,
+		Type:       normativeResult,
+		Description: formDescription(TestShudtownIdentifier,
+			`Ensure that liveness is defined.`),
+		Remediation: `		
```

Comment:
Add the following remediation text:
Ensure that all CNF's pods under test have a liveness **probe** defined.